lib/gruf/rspec.rb

```diff
@@ -45,12 +45,12 @@ module Rspec
   end
 
   config.before(:each, type: :gruf_controller) do
-    define_singleton_method :run_rpc do |method_name, request, &block|
+    define_singleton_method :run_rpc do |method_name, request, options = {}, &block|
```

Comment:
Can we do:

```ruby
define_singleton_method :run_rpc do |method_name, request, active_call_options: {}, &block|
```

That way we don't potentially collide if we need to pass other options later into the other called methods in that block.

Reply:
Makes sense. Also, I think you meant `active_call_options = {}`. I just tested this change against my specs and it works well.

Reply:
I actually meant `active_call_options:` as a keyword argument; I'd like to keep optional arguments as keywords.

Reply:
Got it. I made the changes and also tested them against my specs. Working well!